.github/workflows/build.yaml

```diff
@@ -1,11 +1,9 @@
 ---
 name: build
 on:
-  push:
+  pull_request:
```

Comment:
I changed this from `push` to `pull_reuqest` to align with the behavior I have noticed with some of our Jenkins commit jobs. I believe they build and deploy once the PR is created, so I thought this would be a cleaner workflow. Rather than having a build for every push (since there may be multiple pushes, before a PR is created). Let me know if you think this logic looks good, or I can revert to the original `push` logic.